### ISSUE TITLE
NTR: Rash and pruritis with trial of DEC (Lab: serology & immunological tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8774]
New term request Rash and pruritis with trial of DEC (nadea3_190604153355): Mazzotti test, Mazzotti reaction, diethylcarbamazine

**Comment by pnrobinson** [comment]
see https://pubmed.ncbi.nlm.nih.gov/3194667/

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000801
- label: Positive Mazzotti test
- parents: [Positive skin infectious agent test (HP:0430117)](https://hpo.jax.org/app/browse/term/HP:0430117)
- pmid: PMID:3194667
- def: Locally applied 1.6 percent diethylcarbamazine citrate in Nivea lotion can be used as a diagnostic test for onchocerciasis. Papular reactions are considered as positive test results.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8774
- synonyms:
